pybliometrics/scopus/subject_classifications.py

```diff
@@ -89,7 +89,9 @@ def __init__(self,
         self._refresh = refresh
         self._query = str(query)
         self._view = ''
-        Search.__init__(self, query=query, api='SubjectClassifications', **kwds)
+        # Determine module
+        api = 'SubjectClassifications' if 'scopus' in self.__module__ else 'ScienceDirectSubjectClassifications'
```

Comment:
This looks very complex. Can't we just state the API?

Reply:
Hmm an alternative would be for `SubjectClassifications` to check an optional argument `api` to know which API should be used. But... I find the current implementation better which just checks the name of the file (`__module__`) being used.

Reply:
But currently all classes pass their API (essentially the class) name via this optional keyword. We can change that, and use `self.__module__`, but it should be in a more transparent manner (via a dict mapping or so), and it should be everywhere. Transparency and consistency are also important.

Reply:
Done, with #369 we are consistent across classes.